js/search.js

```diff
@@ -9,12 +9,10 @@ $(document).ready(function() {
           let item = documents[results[r].ref];
           appendString +=
             `<li class="tile">
-              <a class="tile__link" href="${item.url}">
-                <h2 class="tile__title">${item.title}</h2>
-                <p class="tile__callnumber">${item.call_numbers}</p>
-                <p class="tile__authors"><strong>Author(s)</strong>: ${item.author}</p>
-                <p class="tile__date"><strong>Published</strong>: ${item.dates}</p>
-              </a>
+              <a class="tile__title" href="${item.url}">${item.title}</a>
```

Comment:
```suggestion
              <a class="tile__link" href="${item.url}">
                <h2 class="tile__title">${item.title}</h2>
              </a>
```

Reply:
I'm wondering if we can leave this as an h2 in the title because it improves navigation options for sr users

Reply:
Yeah I think I might want to reverse the order of those so the link is inside the h2 though